sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/ConfigurationLabel.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Data.AppConfiguration
+{
+    /// <summary> A label used to group key-values. </summary>
+    public partial class ConfigurationLabel
```

Comment:
Should we consider just `Label` for this?   We don't use `Configuration Label` to describe the fields or selectors.   We call the client method `GetLabelsAsync`.

We should get thoughts from @KrzysztofCwalina before we make any changes.

Reply:
@KrzysztofCwalina - any thoughts on this ?

Reply:
@jorgerangel-msft :  Based on existing patterns in the client, let's use "Label" for this and we'll capture any feedback from Krzysztof in API View.

Reply:
@jsquire - I've renamed the model but doing so threw error `AZC0012` since it was a single word name. I've disabled the error in source based on what I saw was being done in other SDKs, but do let me know if that is not the best practice for our libraries.

Reply:
Thanks, @jorgerangel-msft.   Disabling the analyzer is the right move if we keep the name.   Ultimately, that'll be up to the architects.